lib/hedgelog/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class Hedgelog
-  VERSION = '0.1.12'
+  VERSION = '0.2.1.alpha1'
```

Comment:
For future reference standard naming would be to reset the lower significance number when publishing. Since this is a minor version bump, the patch version should be reset, i.e. `0.2.0.alpha.1` 

https://semver.org/#spec-item-7

Reply:
it was meant to be 0.2.0.alpha1, but when I published I published as just 0.2.0 on accident. So I bumped to 0.2.1.alpha1. to show I was in beta testing, and because of so minor changes I made to pass spec and rubocop